lib/ansible/module_utils/network/nxos/nxos.py

```diff
@@ -742,6 +744,9 @@ def __init__(self, module, cmd_ref_str):
         self._module = module
         self._check_imports()
         self._yaml_load(cmd_ref_str)
+        self.cache_existing = None
```

Comment:
@trishnaguha Fixed this in an open PR.  I needed this patch.

https://github.com/ansible/ansible/pull/59031/files#diff-367b0badce453463ffc33218f1507bdbR1138